packages/babel-plugin-proposal-pipeline-operator/test/fixtures/hack-caret/transform-await-and-arrow-functions/output.js

```diff
@@ -3,9 +3,9 @@ const triple = function (x) {
 };
 
 async function myFunction(n) {
-  var _ref, _ref2, _ref3, _ref4;
+  var _ref;
 
-  return _ref4 = n, (_ref3 = Math.abs(_ref4), (_ref2 = Promise.resolve(_ref3), (_ref = await _ref2, triple(_ref))));
+  return _ref = Math.abs(n), triple(await Promise.resolve(_ref));
```

Comment:
This isn't fully inlined because `Promise.resolve` might be an impure getter. Since Babel assumes unmodified builtins, in Babel 8 we could expand this "implicit assumptions" to also mark this as pure and transform it to
```js
return triple(await Promise.resolve(Math.abs(n)));
```